neuraxle/base.py

```diff
@@ -608,11 +608,8 @@ def get_hyperparams(self, flat=True) -> HyperparameterSamples:
                 hyperparams[k] = hparams
 
         hyperparams = HyperparameterSamples(hyperparams)
-        if flat:
-            hyperparams = hyperparams.to_flat()
-        else:
-            hyperparams = hyperparams.to_nested_dict()
-        return hyperparams
+
+        return hyperparams.to_flat()
```

Comment:
Remove `.toflat()` because we're now guaranteed that the nested steps will themselves return flat things as it was changed everywhere.